### ISSUE TITLE
feature request: Make it possible to use EC main and EC SLP version at the same time. 

### ISSUE THREAD

**Issue by KickassMcFuckyeah** [Electron-Cash/Electron-Cash#1745]
Right now on windows I can not use Electron Cash for BCH and Electron Cash SLP for SLP at the same time. I know what people will say: why don't you just only use EC SLP? 

Because I have lots of old wallet. Some of those are watch wallets from project like BCHpizza.org and some are other projects. Etc etc. I don't want to open those with EC SLP because I get lots of warnings and error messages when I do.

I want to be able to use EC and EC SLP at the same time on windows and linux until the time comes that the two are merged. 

Right now when I open one when the other is open, the other closes.

**Comment by cculianu** [comment]
Yeah the plan is to deprecate/eliminate EC SLP and have all the SLP functionality present in regular Electron Cash.  This should happen soon.

Otherwise the way they are written it's mostly the same codebase and it would be hard to make them run at the same time.  I do agree, it is annoying for users and unfortunate.

**Comment by Natedude** [comment]
Has SLP functionality been added yet?
